### PR TITLE
ENT-10229: Improved the warning text when an added local module contains a policy file with an autorun tag

### PR DESCRIPTION
**Pull request by jakub-nt**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a PR! Maybe @larsewi can review this?

**Review by larsewi — APPROVED** [review]
Great 🚀

File: cfbs/cfbs_config.py [hunk 1]
```diff
@@ -286,13 +286,18 @@ def _handle_local_module(self, module):
             pattern = "%s/**/*.cf" % name
             policy_files = glob.glob(pattern, recursive=True)
 
+        modules_available = [m.get("name", "") for m in self.get("build", [])]
+        is_autorun_enabled = any(
+            m in modules_available for m in ["autorun", "./def.json"]
+        )  # only a heuristic
+
         for file in policy_files:
             if _has_autorun_tag(file):
```
Comment (olehermanse) [inline cfbs/cfbs_config.py:295]: Could `and` here instead of nested `if`s.

Comment (jakub-nt) [reply cfbs/cfbs_config.py:295]: This was on purpose, as the comment at https://github.com/cfengine/cfbs/pull/200/files#diff-16156b18d8d412a896d384b37f3947253ba5f775b6357a7205e0ef533d776d53R301 applies to the first condition, but not the second.

**Pull request merged**